devicemodel/hw/pci/xhci.c

```diff
@@ -1654,15 +1656,23 @@ pci_xhci_alloc_usb_xfer(struct pci_xhci_dev_emu *dev, int epid)
 }
 
 static void
-pci_xhci_free_usb_xfer(struct usb_xfer *xfer)
+pci_xhci_free_usb_xfer(struct pci_xhci_dev_emu *dev, struct usb_xfer *xfer)
 {
 	int i;
 
 	if (!xfer)
 		return;
```

Comment:
You haven't fix the comment on mailing list. Please check !dev also.

Reply:
@ywan170  Check the dev in     "if (dev && dev->dev_ue->ue_free_req)"